### ISSUE TITLE
Store jobids in config to ease subsequent debugging and performance statistics

### ISSUE THREAD

**Issue by lentendu** [lentendu/DeltaMP#42]
In GitLab by @lentendu on Sep 19, 2018, 21:16

at the end of pipeline_master echo all jobid and their corresponding step name stored in variables into a file:
```
echo -e "get\t${get_jobid}#${TECH}_${RAW_EXT}\t${raw_jobid} ... " | tr "#" "\n" > config/jobid
```

**Comment by lentendu** [comment]
In GitLab by @lentendu on Sep 22, 2018, 22:26

mentioned in commit 832f7699b8444565546a9ba2f608230c1fe0f7a2

**Comment by lentendu** [comment]
In GitLab by @lentendu on Sep 24, 2018, 01:33

closed via merge request !5

**Comment by lentendu** [comment]
In GitLab by @lentendu on Sep 24, 2018, 01:33

closed via commit 832f7699b8444565546a9ba2f608230c1fe0f7a2
